plugin.program.bluetoothctl/resources/lib/bluetoothctl.py

```diff
@@ -0,0 +1,155 @@
+from __future__ import annotations
+from typing import Any
+import subprocess
+from subprocess import CompletedProcess
```

Comment:
Just noting this from the other comment.  Rather than using subprocess and call a binary, you should use the dbus API.